### PR TITLE
V2.1 Feat Request

### LINKED ISSUE #17: Dehoist on join/name change

### LINKED ISSUE #18: Join/Leave ID Log

### LINKED ISSUE #20: Duty system

### PR DESCRIPTION
**Pull request by avasilic**
Note: agreed via DMs to make a huge PR vs. smaller ones

### CODE REVIEW

File: src/types/help-roles.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const languages = ["Javascript", "Typescript", "Python", "Rust", "Java", "Kotlin", "CSharp", "CPlusPlus", "C", "Go", "Assembly", "PHP", "Lua", "Swift", "Haskell", "Dart", "Ruby", "Elixir", "Lisp", "Perl", "Scala", "Clojure", "Other"]
+
+let platforms = ["Discordjs", "Discordpy", "HTMLCSS", "MongoDB", "SQL", "React", "Other"]
+
+let helpRoles = [...languages, ...platforms]
+
+export {
```
Comment (tm21cy) [inline src/types/help-roles.ts:5]: These aren't types, so these would be better fit for somewhere like utilities.

Comment (avasilic) [reply src/types/help-roles.ts:5]: Fixed in https://github.com/tm21cy/Bit/pull/13/commits/7ca3cf6d4cb364bbc50b3996f2f69a9434a96702.

**Review by tm21cy — APPROVED** [review]
Will be merged in a development branch and contributed to over the coming days.

**Comment by avasilic** [comment]
Closed #20, #18, #17

**Pull request merged**